client/task-list/tasks/payments/index.js

```diff
@@ -216,28 +230,46 @@ class Payments extends Component {
 		} );
 	}
 
+	getSetupElement( method ) {
+		return cloneElement( method.container, {
+			query: this.props.query,
+			installStep: this.getInstallStep( method ),
+			markConfigured: this.markConfigured,
+			markConfigurationFinished: this.markConfigurationFinished.bind(
```

Comment:
I like the binding of the argument but should it be consistent with `markConfigured`? Can that one be `bind`ed too? (Not a big deal.)

Reply:
That's [not as trivial to change](https://github.com/woocommerce/woocommerce-admin/blob/4cf137ff2d1240478395abf787795740b5cd8939/client/task-list/tasks/payments/klarna.js#L24-L27).